src/dmqmc_data.f90

```diff
@@ -286,6 +292,9 @@ module dmqmc_data
     ! level of the density matrix.
     real(p), allocatable :: excit_dist(:) ! (0:max_number_excitations)
 
+    ! Estimation for RDM-2 (2-body Green Functions)
+    real(p), allocatable :: green(:, :)
+
```

Comment:
The 2-RDM is a four index quantity, could you add a comment about how the elements are stored.